ITensorGPU/test/Project.toml

```diff
@@ -1,10 +1,7 @@
 [deps]
-CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
-Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
-ITensors = "9136182c-28ba-11e9-034c-db9fb085ebd5"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[extras]
-CUDA_Runtime_jll = "76a88914-d11a-5bdc-97e0-2f5a05c973a2"
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+ITensors = "9136182c-28ba-11e9-034c-db9fb085ebd5"
```

Comment:
We should move these back to keep them in alphabetical order.

Reply:
Yes no problem! ( I removed them all to determine if they were the problem with jenkins/necessary for testing).